README.md

```diff
@@ -1,150 +1,8 @@
-# Mount Etna common library
+These JavaScript components are shared across Mount Etna projects. You can install them via NPM into your project with: `npm install --save git+ssh://git@github.com:mountetna/etna-js.git`
```

Comment:
This appears to be out of date here.

Reply:
Also it appears to be overwriting the existing README :) Most of the existing README should move to mountetna.github.io anyway though...